### PR TITLE
4.5 tune transfer

### PR DESCRIPTION
**Pull request by limonspb**
three commits in this PR:
image

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]
Only 5000 lines :smiley:

File: presets/4.5/tune/Street_League_LeadFingers.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#$ TITLE: Street League LeadFingers PIDtoolbox tune
+#$ FIRMWARE_VERSION: 4.4
```
Comment (haslinghuis) [inline presets/4.5/tune/Street_League_LeadFingers.txt]: Should this be 4.5 - seen others in second commit.

File: presets/4.5/tune/minchan_2022_race.txt [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+#$ DESCRIPTION:
+#$ DESCRIPTION: <img src="https://user-images.githubusercontent.com/2925027/198938547-053e2f86-5491-4006-9205-c5eadc6a3df1.png" width="220px" style="display: block; float: left; margin-right: 10px; margin-top: 10px;"/>
+#$ DESCRIPTION:
+#$ DESCRIPTION: This is a 6S 5'' racing tune by MinChan Kim (aka MCK) from 2022 MultiGP Champs, where he took first place,
+#$ DESCRIPTION: proving again that he is the fastest racer in the world.
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: Original tune comes from 4.3 (KAACK mod), but 4.4 tuning parameters are identical to 4.3, except Anti Gravity.
```
Comment (haslinghuis) [inline presets/4.5/tune/minchan_2022_race.txt:18]:
```suggestion
#$ DESCRIPTION: Original tune comes from 4.3 (KAACK mod), but 4.5 tuning parameters are identical to 4.4, except Anti Gravity.
```

File: presets/4.5/tune/Street_League_LeadFingers.txt (hunk 1 above)
Comment (haslinghuis) [reply presets/4.5/tune/Street_League_LeadFingers.txt]: Yeah fixed in commit 3 - was going one by one. So ignore.

File: presets/4.5/tune/minchan_2022_race.txt (hunk 2 above)
Comment (limonspb) [reply presets/4.5/tune/minchan_2022_race.txt:18]: i'll just remove this description line :)

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**